### PR TITLE
weather displaying on cesium

### PR DESCRIPTION
**Pull request by Joe102192**
image

### CODE REVIEW

File: backend/server.ts [hunk 1]
```diff
@@ -12,6 +12,7 @@ const port = process.env.PORT;
 const angularPath = (__dirname.endsWith("dist") ? '/../../' : '/../') + 'frontend/Weather101/dist/weather101/browser/';
 
 app.use('/weather', proxy('https://api.weather.gov'));
+app.use('/nasa', proxy('https://gibs.earthdata.nasa.gov'));
```
Comment (Dalton098) [inline backend/server.ts:15]: doesnt look like this is used, could clean it up

File: frontend/Weather101/src/app/components/dashboard/dashboard.component.scss [hunk 2]
```diff
@@ -0,0 +1,7 @@
+html, body, div[appCesium] {
+    width: 100%;
+    height: 100%;
+    margin: 0;
+    padding: 0;
```
Comment (JoeBartelmo) [inline frontend/Weather101/src/app/components/dashboard/dashboard.component.scss:5]: i am im

**Review by Dalton098 — APPROVED** [review]
WOWIE! besides my comments this looks great joe

File: frontend/Weather101/src/app/directives/cesium.directive.ts [hunk 3]
```diff
@@ -1,5 +1,11 @@
 import { Directive, OnInit, ElementRef } from '@angular/core';
-import { SceneMode, Viewer } from 'cesium';
+import { ImageryLayer, 
+         ImageryProvider,
+         SceneMode,
+         Viewer,
```
Comment (JoeBartelmo) [inline frontend/Weather101/src/app/directives/cesium.directive.ts:5]: O, nicE!

File: frontend/Weather101/src/app/directives/cesium.directive.ts [hunk 4]
```diff
@@ -19,9 +30,19 @@ export class CesiumDirective implements OnInit {
...
+    this.viewer = new Viewer(this.el.nativeElement, {
+      sceneMode: SceneMode.SCENE2D
+    });
 
-}
\ No newline at end of file
+    const layer = new ImageryLayer(new WebMapServiceImageryProvider({
+      url: "/mesonet/cgi-bin/wms/nexrad/n0r.cgi?",
+      layers: "nexrad-n0r",
+      credit: "Radar data courtesy Iowa Environmental Mesonet",
```
Comment (JoeBartelmo) [inline frontend/Weather101/src/app/directives/cesium.directive.ts:40]: WOW!34

File: frontend/Weather101/src/app/directives/cesium.directive.ts (hunk 3 above)
Comment (JoeBartelmo) [reply frontend/Weather101/src/app/directives/cesium.directive.ts:5]: NICE

**Review by JoeBartelmo — APPROVED** [review]

**Pull request merged**